src/main/java/com/CezaryZal/api/body/BodySizeController.java

```diff
@@ -13,50 +16,45 @@
 @RequestMapping("/api/body")
 public class BodySizeController {
 
-    private BodySizeService bodySizeS;
+    private BodySizeService bodySizeService;
 
-    public BodySizeController(BodySizeService bodySizeS) {
-        this.bodySizeS = bodySizeS;
+    public BodySizeController(BodySizeService bodySizeService) {
+        this.bodySizeService = bodySizeService;
     }
 
-    @ApiOperation(value = "This will get a `BodySize by id", notes = "In this method you will receive a body measurement by id")
+    @ApiOperation(value = "This will get a `BodySize` by id", notes = "In this method you will receive a body measurement by id")
     @GetMapping("/{id}")
-    public BodySize getBodyById (@PathVariable Long id){
-        return bodySizeS.getBodyById(id);
+    public BodySizeDto getBodySizeDtoById(@PathVariable Long id) {
```

Comment:
o to chodzi, endpoint ma zwracać obiektDTO, nie encje

Reply:
ok